schemas/data/src/addition/ResultAnalyticalMethodContext.json

```diff
@@ -1 +1 @@
-["PROPRIETARY", "SM", "VMV"]
\ No newline at end of file
+["PROPRIETARY", "SM", "VMV", "Seal", "SM23ED"]
```

Comment:
`Seal` is already in WQX, does not need to be included
Has `SM23ED` been submitted to WQX? Related to `SM`?